benchmark/python/benchmark_hail/run/utils.py

```diff
@@ -142,51 +173,73 @@ def initialize(args):
 
     # make JVM do something to ensure that it is fresh
     hl.utils.range_table(1)._force_count()
+    logging.getLogger('py4j').setLevel(logging.CRITICAL)
+    logging.getLogger('py4j.java_gateway').setLevel(logging.CRITICAL)
+
+
+def run_with_timeout(b, max_time):
+    with timeout_signal(max_time):
+        try:
+            return timeit.Timer(b.run).timeit(1), False
+        except Py4JError as e:
+            if _timeout_state:
```

Comment:
I need this global variable to catch py4jerrors, which wrap the real exception in a way that makes it very difficult to see the original cause.